### DISCUSSION TITLE
Responsive for below 640 px

### DISCUSSION THREAD

**Discussion by mohitTecorb** [tailwindlabs/tailwindcss#11778] [Ideas]
Please add more classes to make responsive , below 640 px screen .

**Comment by andymccray** [comment]
Any Tailwind class _without_ a responsive modifier (e.g. without `sm:` or similar) will work below 640px, e.g. it'll take effect _until the window hits 640px_, then the small modifier will take over. If you don't add any responsive modifier classes then the naked class will persist.

If you need more granular control before that first breakpoint you can extend your theme to add additional breakpoints, e.g. one that kicks in at 320px.

```js
module.exports = {
  theme: {
    extend: {
      screens: {
        'xs': '320px'
      }
    }
  }
}
```

You can also [totally customise the approach to responsive design](https://tailwindcss.com/docs/responsive-design#customizing-your-theme).

Hopefully that helps!

**Reply by mohitTecorb** [reply]
yeah .. 
Thanks @andymccray  for your response, 
I know about this customise screen class. 
am asking about  utility class for that.

**Reply by mybearworld** [reply]
@mohitTecorb For most use cases being able to customize 0px-640px is just fine and you don't need to specify it further.

**Reply by 2345david** [reply]
broder no me funciona por que?
